diff --git a/node_modules/force-graph/dist/force-graph.module.js b/node_modules/force-graph/dist/force-graph.module.js
index 4c312a8..025b93b 100644
--- a/node_modules/force-graph/dist/force-graph.module.js
+++ b/node_modules/force-graph/dist/force-graph.module.js
@@ -1171,8 +1171,8 @@ var forceGraph = Kapsule({
     container.appendChild(toolTipElem); // Capture mouse coords on move
 
     var mousePos = {
-      x: -1e12,
-      y: -1e12
+      x: 0,
+      y: 0
     };
     state.canvas.addEventListener('mousemove', function (ev) {
       // update the mouse pos
